backend/app/usecase/shortlink/creator_test.go

```diff
@@ -226,13 +202,10 @@ func TestShortLinkCreatorPersist_CreateShortLink(t *testing.T) {
 				riskDetector,
 			)
 
-			_, err = shortLinkRepo.GetShortLinkByAlias(testCase.shortLink.Alias)
```

Comment:
Deleted this short link repo check because it produced a false positive for alias_exists test. Not sure if it's really needed.

Reply:
To add a bit more information here: It produced a false positive because existing alias is now passed in alongside the input object, and `GetShortLinkByAlias` checks that.

I think checking for alias existing is a good idea for all other tests, because every other test besides the alias exists test should not have any aliases and we should check that for test correctness.

Reply:
I will add this check back in, but have some sort of way to bypass it only for the `alias_exists` test.

Reply:
@byliuyang Just wanted to mark this as unresolved briefly. So do you think `shortLinkRepo.GetShortLinkByAlias` on line 229 should be added back in? Or is the `userShortLinkRepo.HasMapping` check below it sufficient for checking short link existence?

Reply:
Resolved this by adding a `shouldAliasExist` bool in test case. Only `alias_exists` test will have this set as true since alias is expected to be there for that test. All other tests should ensure that alias does not already exist, as a precondition check.